apache/zeppelin
# [ZEPPELIN-2587] allow logging in if you're anonymous

### What is this PR for?

The login button is not showing (since 0.7.1) if you allow anonymous access. This shows it again.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN/ZEPPELIN-2587

### How should this be tested?
- Start Zeppelin with anonymous support
- The login button should now show on the bar

### Questions:
* Does the licenses files need update?
No
* Is there breaking changes for older versions?
No
* Does this needs documentation?
No
